src/scene/scene_forwardrenderer.js

```diff
@@ -368,6 +368,8 @@ pc.extend(pc, function () {
             device.setScissor(x, y, w, h);
 
             device.clear(camera.getClearOptions());
+
+            if (cullBorder) device.setScissor(1, 1, pixelWidth-2, pixelHeight-2);

```

Comment:
I honestly don't see a point of {} here. It may make sense only when (cullBorder) is executable code, in which we can better see which line caused a bug.